packages/checkbox/checkbox.scss

```diff
@@ -0,0 +1,89 @@
+@import "~@fremtind/jkl-core/build/scss/variables/colors";
+@import "~@fremtind/jkl-core/build/scss/functions";
+
+$checkbox-size: 1.5rem;
+
+.jkl-checkbox {
+    align-items: center;
+    cursor: pointer;
+    display: inline-flex;
+    padding: rem(5px);
+
+    &__input {
+        -webkit-appearance: none;
+        background-color: $snøhvit;
+        border: solid rem(1px) $svart;
+        height: $checkbox-size;
+        margin-right: 0.5rem;
```

Comment:
Remember to use the spacing scale. `$component-spacing--small` is the equivalent here

Reply:
I thought the component scale was mainly for vertical spacing - not horizontal?

Reply:
I'm thinking that the layout scale is for vertical, but the component spacing is for both? Sounds like we should have a chat with design about this one..